pages/pydata.md

```diff
@@ -0,0 +1,30 @@
+title: PyData Track
+---
+
+This year, for the first time, the teams at PyConUK and [PyData London](http://london.pydata.org/) are joining forces to deliver a data science focussed track at the conference. We will be hosting a number of talks and workshops about data science at the the conference, and we are looking for members of the PyData community to submit proposals to the [conference CFP](https://hq.pyconuk.org/cfp/proposals/new/).
```

Comment:
nit: the conference name is “PyCon UK”, not “PyConUK”

(@inglesp: I’m sure we used to have CI checks for this?)